src/style-sheets/generic/_button.scss

```diff
@@ -1,32 +1,42 @@
 .button {
-    width: 189px;
-    height: 49px;
-    border-radius: 4px;
-    border-width: 0;
+  width: 189px;
+  height: 49px;
+  border-radius: 4px;
+  border-width: 0;
 
-    font-size: 1.25rem;
-    cursor: pointer;
+  font-size: 1.25rem;
+  cursor: pointer;
 }
 
 .primary {
-    background-color: $olLTAccent;
-    color: $white;
-    font-family: $olBaseFont;
+  background-color: $olLTAccent;
+  color: $white;
+  font-family: $olBaseFont;
+
+  &:hover {
+    background-color: $olLTPrimaryBtnHover;
+  }
+
+  &:active {
+    background-color: $olLTPrimaryBtnClicked;
+    color: $olLTPrimaryBtnTxtClicked;
+    border-radius: 4px;
```

Comment:
we have this same border-radius declared in the .button rule above, so remove please. The button rule and the .primary rule are supposed to be used together (button being the base rule). I should probably add some documentation in the file to lay that out so sorry about not having that explanation.

Reply:
nah it's pretty self-explanatory I just musta missed it first time round.